libs/LedKit/source/animations/Heartbeat.cpp

```diff
@@ -0,0 +1,163 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+// ? LCOV_EXCL_START - Exclude from coverage report
+
+#include "Heartbeat.h"
+
+#include "LogKit.h"
+using namespace std::chrono;
+namespace leka::led::animation {
+
+void Heartbeat::setLeds(interface::LED &ears, interface::LED &belt)
+{
+	_belt = &belt;
+}
+
+auto Heartbeat::isRunning() -> bool
+{
+	return _running;
+}
+
+void Heartbeat::start()
+{
+	if (_belt == nullptr) {
+		return;
+	}
+
+	turnLedBlack();
+	_step	 = 0;
+	_stage	 = 1;
```

Comment:
```suggestion
	_stage	 = 0;
```
Meme remarque, décrementer tout de 1.